src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -84,6 +84,18 @@
       <TestCopyLocal Include="@(OverridenRuntimeContent)" />
      </ItemGroup>
 
+    <ItemGroup Condition="'$(CoreLibRoot)' != ''">
```

Comment:
No, but right now it's why I need it. Perhaps I should have put it in separately but it doesn't seem worth separating now.